src/yoroi-wallets/cardano/ShelleyWallet.ts

```diff
@@ -264,7 +263,10 @@ export class ShelleyWallet implements WalletInterface {
     const rewardAddressHex = await deriveRewardAddressHex(accountPubKeyHex, networkId)
     const apiUrl = getCardanoNetworkConfigById(networkId).BACKEND.API_ROOT
     const utxoManager = await makeUtxoManager({storage: storage.join('utxoManager/'), apiUrl})
-    const transactionCache = await TransactionCache.create(storage.join('txs/'))
+    const transactionManager = await makeTransactionManager(
+      storage.join('txManager/'),
+      getCardanoNetworkConfigById(networkId).BACKEND,
```

Comment:
why move this here?

Reply:
Discussed here: https://github.com/Emurgo/yoroi-mobile/pull/2306#discussion_r1088177239